package.json

```diff
@@ -25,8 +26,9 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "bower": "^1.8.0",
```

Comment:
For hybrid, we should remove this `bower` dependency and use polymer cli to install the bower dependencies... `"postinstall": "polymer install --variants"`.